ci/cases/pr/ATM_cold.yaml

```
@@ -0,0 +1,11 @@
+untar:
+    tarball : "{{ ATARDIR }}/{{ cycle_YMDH }}/atm_cold.tar"
```

This should be `FETCHDIR` now

```suggestion
    tarball : "{{ FETCHDIR }}/{{ cycle_YMDH }}/atm_cold.tar"
```

---

Done